aspnet/publishing/iis.rst

```diff
@@ -54,6 +54,27 @@ For more information on the ASP.NET Core Module, including configuration of the
 Application configuration
 -------------------------
 
+Enabling the `IISIntegration` service
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Ensure that the `UseIISIntegration()` is injected:
```

Comment:
I'd re-phrase to something along the lines of ...

> Incorporate IIS Integration middleware to the application by adding the `.UseIISIntegration()` extension method to `WebHostBuilder`.

... that's just a suggestion. Also, be sure to drop the colon in favor of a period at the end of the sentence.